roles/oneview_server_profile/README.md

```diff
@@ -0,0 +1,37 @@
+## oneview_server_profile
+Manage OneView Server Profile resources.
+
+#### Synopsis
+ Provides an interface to manage Server Profile resources. Can create, update, and delete.
+
+#### Requirements (on the host that executes the module)
+  * hpeOneView >= 5.8.0
```

Comment:
our recent release version is 5.4.0. Please change this to 5.4.0

Reply:
done